actions/chats.js

```diff
@@ -100,7 +115,7 @@ const socketRemoveUserFromChat = (socket, data) => {
 }
 
 export const fetchChats = () => {
-    return async (dispatch, getState) => {
```

Comment:
Couple different instances of getState param but didn't look like it was being used at all, so I removed it.